generate/run.py

```diff
@@ -1,16 +1,16 @@
+import argparse
+import json
 import os
 import sys
-import json
-import argparse
 from os.path import join, abspath, exists
 
+from binder.core import Generator
+
 # If running outside this folder we need to add this to the syspath
 BINDER_ROOT = os.path.dirname(os.path.dirname(__file__))
 if BINDER_ROOT not in sys.path:
     sys.path.append(BINDER_ROOT)
 
-from binder.core import Generator
```

Comment:
I think pycharm re-ordering the imports here is why it's failing now. 

Modifying the sys path is a hack anyways so maybe there's a better way to do this.